### DISCUSSION TITLE
Good optimizer for vanilla mc with many players?

### DISCUSSION THREAD

**Discussion by 7quit** [YouHaveTrouble/minecraft-optimization#104] [Show and tell]
Good evening, is all the optimisation also good for a vanilla Minecraft server?
Because I've been looking for a server optimisation on the net for a long time.

**Comment by YouHaveTrouble** [comment]
On vanilla you're very limited in things you can do, just server.properties basically.
If you want pure vanilla server without bukkit changes I strongly suggest running fabric with performance mods like lithium.
